phantom-dsl/src/test/scala/com/outworkers/phantom/builder/primitives/PrimitiveColumnRecipesTest.scala

```diff
@@ -67,6 +67,7 @@ class PrimitiveColumnRecipesTest extends PhantomSuite {
   }
 
   it should "retrieve an empty ingredients set" in {
+    import com.outworkers.util.macros.debug.Options.ShowTrees
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://app.codacy.com/manual/flavian/phantom/pullRequest?prid=3695041)